src/server/templates/practices.njk

```diff
@@ -7,7 +7,7 @@
 
   <div class="page-section">
     <div class="reading-width">
-      <p>This is a trial service. It's currently only available for these GP practices:</p>
+      <p class="pad-bottom20">This is a trial service. It's currently only available for these GP practices:</p>
```

Comment:
Classes that set a specific amount of spacing should be avoided. See comments above.